tests/basic/pool_function_test.go

```diff
@@ -151,422 +170,422 @@ var _ = Describe("{PoolExpandSmoky}", func() {
+//				fmt.Sprintf("Timed out waiting for px to come up on node [%s]", storageNode.Name))
+//		})
+//
+//		Step("Ensure pool is up and running", func() {
+//			// Ensure pool is up and running
+//			poolToResize = getStoragePool(poolIDToResize)
+//			// Ensure poolToResize is not nil
+//			dash.VerifyFatal(poolToResize != nil, true, "Pool is up and running after restart")
+//			verifyPoolSizeEqualOrLargerThanExpected(poolIDToResize, targetSizeGiB)
+//		})
+//	})
+//
+//	It("Initiate pool expansion using add-drive and restart PX", func() {
+//		StartTorpedoTest("PoolExpandAddDiskAndPXRestart",
+//			"Initiate pool expansion using add-drive and restart PX", nil, testrailID)
+//
+//		Step("Select a pool that has I/O and expand it by 100 GiB with add-disk type. ", func() {
+//			originalSizeInBytes = poolToResize.TotalSize
+//			targetSizeInBytes = originalSizeInBytes + 100*units.GiB
+//			targetSizeGiB = targetSizeInBytes / units.GiB
+//			log.InfoD("Current Size of the pool %s is %d GiB. Trying to expand to %v GiB with type add-disk",
+//				poolIDToResize, poolToResize.TotalSize/units.GiB, targetSizeGiB)
+//			triggerPoolExpansion(poolIDToResize, targetSizeGiB, api.SdkStoragePool_RESIZE_TYPE_ADD_DISK)
+//		})
+//
+//		Step("Wait for expansion to start and reboot node", func() {
+//			err := WaitForExpansionToStart(poolIDToResize)
+//			log.FailOnError(err, "Timed out waiting for expansion to start")
+//			err = Inst().V.RestartDriver(*storageNode, nil)
+//			log.FailOnError(err, fmt.Sprintf("Error restarting px on node [%s]", storageNode.Name))
+//			err = Inst().V.WaitDriverUpOnNode(*storageNode, addDriveUpTimeOut)
+//			log.FailOnError(err, fmt.Sprintf("Timed out waiting for px to come up on node [%s]", storageNode.Name))
+//		})
+//
+//		Step("Ensure pool has been expanded to the expected size", func() {
+//			resizeErr := waitForOngoingPoolExpansionToComplete(poolIDToResize)
+//			dash.VerifyFatal(resizeErr, nil, "Pool expansion does not result in error")
+//			verifyPoolSizeEqualOrLargerThanExpected(poolIDToResize, targetSizeGiB)
+//		})
+//	})
+//})
+//
+//var _ = Describe("{PoolExpandResizeInvalidPoolID}", func() {
+//
+//	var testrailID = 34542946
+//	// testrailID corresponds to: https://portworx.testrail.net/index.php?/tests/view/34542946
+//
+//	BeforeEach(func() {
+//		StartTorpedoTest("PoolExpandResizeInvalidPoolID",
+//			"Initiate pool expansion using invalid Id", nil, testrailID)
+//	})
+//
+//	AfterEach(func() {
+//		EndTorpedoTest()
+//	})
+//
+//	stepLog := "Resize with invalid pool ID"
+//	It(stepLog, func() {
+//		log.InfoD(stepLog)
+//		// invalidPoolUUID Generation
+//		invalidPoolUUID := uuid.New().String()
+//
+//		// Resize Pool with Invalid Pool ID
+//		stepLog = fmt.Sprintf("Expanding pool on Node UUID [%s] using auto", invalidPoolUUID)
+//		Step(stepLog, func() {
+//			resizeErr := Inst().V.ExpandPool(invalidPoolUUID, api.SdkStoragePool_RESIZE_TYPE_AUTO, 100, true)
```

Comment:
Why remove this test? We can just update the type to RESIZE.

Reply:
Adding back!